src/Azure.Functions.Cli/Common/Utilities.cs

```diff
@@ -49,7 +49,7 @@ internal static void PrintVersion()
         {
             ColoredConsole
                 .WriteLine($"\nAzure Functions Core Tools")
-                .WriteLine($"Core Tools Version:       {Constants.CliDetailedVersion}".DarkGray())
+                .WriteLine($"Core Tools Version:       {Constants.CliDetailedVersion + (Environment.Is64BitProcess ? " (64-bit)" : " (32-bit)")}".DarkGray())
```

Comment:
Tools like VS Code parse the version information. This can potentially break the integration.

cc:@ejizba

Reply:
Thanks for the ping, but I don't think this would affect us. We only care about `func --version` which [seems to use](https://github.com/Azure/azure-functions-core-tools/blob/3a59efba0cbf74dfaf1911de5f073df97259d9f3/src/Azure.Functions.Cli/ConsoleApp.cs#L217) `Constants.CliVersion` instead of this `PrintVersion` method

Reply:
Alternatively, could add a new line in PrintVersion method, which would be less risky of breaking something. 

+ .WriteLine($"Core Tools Architecture:       {(Environment.Is64BitProcess ? " 64-bit" : "32-bit")}".DarkGray())

Though, hopefully nobody is relying on parsing data from a large startup text block. I installed the core tools from a package installer and then it took much more effort that I thought it would to determine which version had been installed (32 or 64  bit), which led to my PR.

Reply:
@JohnCarmichael3000, Can you rebase your branch and sign the license/cla?

Reply:
Great, thanks. I was thinking it was something in my end I hadn't completed. :)